### ISSUE TITLE
RFE: Show player units (including portraits and mini-damage doll), grouped by force, along the top of 'action bar' rather than on the side of the screen

### ISSUE THREAD

**Issue by UlyssesSockdrawer** [MegaMek/megamek#3723]
### Environment
N/A

### Description
Currently, the player's units are shown on the top right of the screen in vertical descending order, and are organised by their ID/order in the lobby.

The information shown includes a small health bar and the unit's sprite.

Perhaps as an optional change, it would be nice to have the pilot's portrait, callsign (if applicable), and the unit's damage doll show instead of the sprite. This would provide an at a glance 'who's who' of the player's units, where damage has been suffered, and who is piloting.

If these could also be grouped by the 'force' the unit is attached to (based on the data from the force organisation view in 49+ versions of Megamek), this would be an additional help in understanding 'who is who' in large battles.

The scroll feature from the top right could be maintained for very large forces. Where no portrait is used, this could be left blank, pull the default portrait (head with question mark) and simply show the damage doll.

Pilot hits could also be shown if graphics space was available, using a series of dots that either filled up, or depleted as damage was taken.

### Files
Example image of idea from old MechCommander Gold game (screenshot from http://www.thegameengine.org/wp-content/uploads/2012/12/mech_commander_gold_1.jpg)

![image](https://user-images.githubusercontent.com/88290873/174148218-cce05c98-d63c-4520-9138-1a34cad25c74.png)

**Comment by BLR-IIC** [comment]
I really like this as an option.  I feel like the current vertically-aligned unit list is too compacted.  One could probably see a company's worth of units or more without scrolling if the unit icon list could be re-positioned to the top of the screen instead.

**Comment by pakfront** [comment]
I like this as well - I think some experimentation to get a good overall layout, including buttons and the weapons panel, would be in good.

**Comment by JonhyBurner** [comment]
I like the idea of changing the layout to a top horizontal one. The showing of the armor doll and the pilot image should be optional, but it's also a good idea.
Maybe the pilot image could be filled with a dark grey layer for every hit, with 5 hits leading to a fiiled pilot image.
I would like to add that keeping destroyed units in view would be good to assess current fighting ability of the force (something the simple BV percentage doesn't show properly). In this case, dead pilots would have a different overlay, perhaps a red cross over it.

**Comment by UlyssesSockdrawer** [comment]
Definitely - I think if units remained on the UI after fleeing the battle or being destroyed it would be very helpful. For dead or ejected pilots, MechCommander used a skull icon that replaced the portrait, or a parachute icon respectively, but a red cross or parachute overlay would be pretty neat.

**Comment by UlyssesSockdrawer** [comment]
Good point Simon - MechCommander had unique damage dolls for each 'mech, but had a much smaller roster. Using the MM generic damage doll + either pilot portrait or sprite would be a good option to have.

This is one of those RFEs I'd love to do, but have no idea how to do myself.
